### DISCUSSION TITLE
404 file not found

### DISCUSSION THREAD

**Discussion by Vardhan077** [community/community#103107] [Actions]
### Select Topic Area

Bug

### Body

![Screenshot 2024-02-02 154609](https://github.com/community/community/assets/132035215/c9858898-69d0-4c01-a33f-33ccbb4127fb)

I can see my output in localhost, but I can only main page and not the other pages(Routes) after publishing through gh-pages 

my repo link ---> https://github.com/Vardhan077/ecomWebsite

can anyone help me with this

**Comment by ghost** [comment]
Answered it in the specific issue you created in you repo I thought answering there might be more accessible to you

**Reply by ghost** [reply]
hope i was helpful if nott im sorry
